cypress/e2e/TemplateDapp/SCTransaction.cy.ts

```diff
@@ -0,0 +1,31 @@
+/// <reference types="cypress" />
+import { scTransaction } from './SCActions';
+import { SCTransactionData,scSelectors } from '../TemplateDapp/SCTransactionData';
+
+describe('Smart Contract Transactions', () => {
+  before(() => {
+    cy.visit('/');
+    cy.getSelector(scSelectors.loginBtn).click();
+    cy.getSelector(scSelectors.webWalletLogin).click();
+    cy.login();
+  });
+  it('Ping & Pong actions', () => {
+    //The condtionals will check the actual status of the tranzaction and proceed with the properly actions.
```

Comment:
tran`s`action typo

and proceed with the properly actions. --> a bit  vague. Either exclude it or rewrite it to something more explicit